apps/cli-gui/src/app/app.routes.ts

```diff
@@ -4,15 +4,11 @@ export const APP_ROUTES: Routes = [
   {
     path: '',
     children: [
-      {
-        path: '**',
-        redirectTo: 'generators',
-      },
       {
         path: 'generators',
-        loadComponent: () =>
-          import('@angular-cli-gui/generators').then(
-            (m) => m.GeneratorsComponent
+        loadChildren: () =>
+          import('./generators/generators.routes').then(
+            (m) => m.GENERATOR_ROUTES
           ),
```

Comment:
FYI if you add `export default GENERATOR_ROUTES;` in generator.routes you could do
```suggestion
        loadChildren: () => import('./generators/generators.routes'),
```